flink-table/flink-table-planner-blink/src/test/java/org/apache/flink/table/planner/functions/aggfunctions/AggFunctionTestBase.java

```diff
@@ -100,15 +100,15 @@ public void testAggregateWithMerge()
 				List<T> inputValues = inputValueSets.get(i);
 				T expected = expectedResults.get(i);
 				// equally split the vals sequence into two sequences
-				Tuple2<List<T>, List<T>> splitValues = splitValues(inputValues);
+				Tuple2<List, List> splitValues = splitValues(inputValues);
 				List<T> firstValues = splitValues.f0;
 				List<T> secondValues = splitValues.f1;
+
 				// 1. verify merge with accumulate
+				ACC acc = accumulateValues(firstValues);
 				List<ACC> accumulators = new ArrayList<>();
 				accumulators.add(accumulateValues(secondValues));
 
-				ACC acc = accumulateValues(firstValues);
-
```

Comment:
I update I modified it because we need to split valueList which type is `T`, and we also need to split orderList which type is `Long`. I think it should be the same because of type erasure. 
But it's really not elegant, or we can use `Object` type  for list element for splitValues method？